jte/src/main/java/gg/jte/compiler/TemplateCompiler.java

```diff
@@ -64,13 +64,43 @@ public void cleanAll() {
     @Override
     public List<String> generateAll() {
         LinkedHashSet<ClassDefinition> classDefinitions = generate(codeResolver.resolveAllTemplateNames());
+        generateNativeResources(classDefinitions);
         return classDefinitions.stream().map(ClassDefinition::getSourceFileName).collect(Collectors.toList());
     }
 
     @Override
     public List<String> precompileAll() {
         return precompile(codeResolver.resolveAllTemplateNames());
     }
+    
+    private void generateNativeResources(LinkedHashSet<ClassDefinition> classDefinitions) {
+        if (config.resourceDirectory == null) {
```

Comment:
Could you add a setting `generateNativeResources` that controls this explicitly?

Besides that the PR looks good to merge 👍